golem_messages/cryptography.py

```diff
@@ -319,12 +320,12 @@ def ecies_decrypt(self, data, shared_mac_data=b''):
     encrypt = ecies_encrypt
     decrypt = ecies_decrypt
 
-    def sign(self, data):
+    def sign(self, data):  # pylint: disable=arguments-differ
```

Comment:
Is this needed? The number of arguments is the same. It's just that arguments have different names.